README.md

```diff
@@ -79,6 +79,13 @@ you can run tests by running this line in the terminal:
 
```

Comment:
- [ ] To make your documentation looks professional and descriptive, I think it would be better if all the contributors of the current project are present in the `Authors` section of your README.